saleor/graphql/app/sorters.py

```diff
@@ -22,3 +22,20 @@ class Meta:
         doc_category = DOC_CATEGORY_APPS
         sort_enum = AppSortField
         type_name = "apps"
+
+
+class AppEventSortField(BaseEnum):
+    CREATION_DATE = ["created_at", "pk"]
```

Comment:
I think that it should be `CREATED_AT`. At least for the `OrderSort` we deprecate the `CREATION_DATE` in favor of `CREATED_AT`